src/lib/watchNotification/watchNotification.ts

```diff
@@ -62,12 +62,22 @@ export async function shouldUserBeNotified(
 		}

```

Comment:
Uhh what about this loop? Did you mean to remove it? I think you reproduced it below but didn't remove the bad forEach version

Reply:
nice callout- fixed